frontend/src/pages/ResumeFeedbackPage.tsx

```
@@ -1,5 +1,6 @@
 import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
+import { useParams } from "react-router-dom";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'useParams'. Consider to delete it or rename it.

'useParams' is defined here:


(lint/suspicious/noRedeclare)